### ISSUE TITLE
filter out comments

### ISSUE THREAD

**Issue by genebean** [NiklasRosenstein/pydoc-markdown#108]
I have a block like this at the top of each file and its being included in the markdown: 

```
# -*- coding: utf-8 -*-
# Copyright (c) 2014 Jim Kemp <kemp.jim@gmail.com>
# Copyright (c) 2017 Gene Liverman <gene@technicalissues.us>
# Distributed under the MIT License (https://opensource.org/licenses/MIT)
```

Can you filter this out automatically?

**Comment by NiklasRosenstein** [comment]
Hey @genebean , Which version of Pydoc-markdown are you currently using? The one from the `develop` branch or 2.x from PyPI?

I think we could add an option to the Python parser to ignore single-line comments when looking for the docstrings of a module!

**Comment by genebean** [comment]
Thanks! I am using v3 / develop

**Comment by NiklasRosenstein** [comment]
@genebean You can set the `treat_singleline_comment_blocks_as_docstrings` option on the `python` loader to False to ignore single-line block comments when looking for docstrings.

```yml
loaders:
  - type: python
    treat_singleline_comment_blocks_as_docstrings: false
# ...
```

**Comment by genebean** [comment]
Thank you!

**Comment by NiklasRosenstein** [comment]
FYI I just published pydoc-markdown v3.0.0 to PyPI so you won't have to go through installing from GitHub anymore :)
